content/docs/2.7/scalers/cpu.md

```diff
@@ -30,11 +31,32 @@ triggers:
 - `value` - Value to trigger scaling actions for:
 	- When using `Utilization`, the target value is the average of the resource metric across all relevant pods, represented as a percentage of the requested value of the resource for the pods.
 	- When using `AverageValue`, the target value is the target value of the average of the metric across all relevant pods (quantity).
+- `containerName` - Set this to target the specific container rather than the entire pod. Defaults to empty if not specified.
```

Comment:
we should mention that this is supported from k8s 1.20 and maybe a link to the upstream docs: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#container-resource-metrics

Reply:
Updated the doc